src/__tests__/__snapshots__/theme.js.snap

```diff
@@ -12,66 +12,67 @@ Object {
   "colors": Object {
     "black": "#000",
     "blue": "#007aff",
-    "blue0": "#cce4ff",
-    "blue1": "#99caff",
-    "blue2": "#66afff",
-    "blue3": "#3395ff",
+    "blue0": "#cdf",
+    "blue1": "#007aff",
+    "blue2": "#049",
     "blues": Array [
-      "#cce4ff",
-      "#99caff",
-      "#66afff",
-      "#3395ff",
+      "#cdf",
+      "#007aff",
+      "#049",
     ],
-    "gray": "#596b7d",
-    "gray0": "#dee1e5",
-    "gray1": "#bdc4cb",
-    "gray2": "#9ba6b1",
-    "gray3": "#7a8997",
+    "darkBlue": "#049",
+    "darkGray": "#364049",
+    "darkGreen": "#060",
+    "darkOrange": "#950",
+    "darkPurple": "#407",
+    "darkRed": "#800",
+    "gray": "#687B8E",
+    "gray0": "#f0f2f4",
+    "gray1": "#687B8E",
+    "gray2": "#364049",
     "green": "#0a0",
-    "green0": "#cceecc",
-    "green1": "#99dd99",
-    "green2": "#66cc66",
-    "green3": "#33bb33",
+    "green0": "#cec",
+    "green1": "#0a0",
+    "green2": "#060",
     "greens": Array [
-      "#cceecc",
-      "#99dd99",
-      "#66cc66",
-      "#33bb33",
+      "#cec",
+      "#0a0",
+      "#060",
     ],
+    "lightBlue": "#cdf",
+    "lightGray": "#f0f2f4",
+    "lightGreen": "#cec",
+    "lightOrange": "#fec",
+    "lightPurple": "#ecf",
+    "lightRed": "#fcc",
     "orange": "#f90",
-    "orange0": "#ffebcc",
-    "orange1": "#ffd699",
-    "orange2": "#ffc266",
-    "orange3": "#ffad33",
```

Comment:
This will probably be a "breaking" change for anyone using a `3` color. Should we map these to `2` and add a prop-type warning that it will be deprecated?